use-shopping-cart/core/store.js

```diff
@@ -0,0 +1,60 @@
+import { configureStore } from '@reduxjs/toolkit'
+import { reducer, actions, initialState } from './slice'
+import { filterCart, formatCurrencyString } from '../utilities/old-utils'
+import { handleStripe } from './stripe-middleware'
+import { handleWarnings } from './warning-middleware'
+import {
+  persistStore,
+  persistReducer,
+  FLUSH,
+  REHYDRATE,
+  PAUSE,
+  PERSIST,
+  PURGE,
+  REGISTER
+} from 'redux-persist'
+import persistStorage from 'redux-persist/lib/storage'
+
+const createNoopStorage = () => {
+  return {
+    getItem(_key) {
+      return Promise.resolve(null)
+    },
+    setItem(_key, value) {
+      return Promise.resolve(value)
+    },
+    removeItem(_key) {
+      return Promise.resolve()
+    }
+  }
+}
+
+const storage =
+  typeof window === 'undefined' ? createNoopStorage() : persistStorage
```

Comment:
Don't forget, we have a variable for this already: `isClient` in the file `utilities/SSR.js`.